### ISSUE TITLE
Sequencer skips track without feedback to user

### ISSUE THREAD

**Issue by rsjtdrjgfuzkfg** [nightingale-media-player/nightingale-hacking#364]
For one specific flac track (copyrighted, so I cannot attach it here), the sequencer will skip to the next track once a specific position is reached (reproduceable). Playback before and after said position work as expected, and other media players play the file just fine (avconv/ffmpeg, vlc, totem (which should use gstreamer)).

There is no error console output, no unsupported format or invalid file window or output to stderr/stdout.

I certainly do not have the time required to look into the issue at all (I'd assume the file is a tiny bit broken but modern libs can handle that), this is merely the report that we have media core issues that do not result in any user notification, which shouldn't happen.

Debian Jessie x64 / sb-trunk-oldxul from Nov 2014; but I cannot recall any media core changes since then, I'll try to remember to test on a recent build once I am a tiny bit less busy irl.
##

**Comment by freaktechnik** [comment]
This honestly sounds very similar to #60.

**Comment by rsjtdrjgfuzkfg** [comment]
@freaktechnik I don't think so, as it is only one very specific point in the track that causes issues, and that independently on how you get there (skip or regular play).

**Comment by JazzMaster** [comment]
it might be hitting hidden web code or something in the wrong place. try checking the file with mp3diags.
